src/domains/gps-common/00_monitor.tf

```diff
@@ -21,3 +21,36 @@ data "azurerm_monitor_action_group" "email" {
   resource_group_name = var.monitor_resource_group_name
   name                = local.monitor_action_group_email_name
 }
+
+##Alert
+resource "azurerm_monitor_scheduled_query_rules_alert" "payments_gpd_inconsistency_error" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = format("%s-gpd-payments-api-alert", var.env_short)
+  resource_group_name = azurerm_resource_group.gps_rg.name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.name, data.azurerm_monitor_action_group.slack.id]
+    email_subject          = "[Payments] call GPD payment position error"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = data.azurerm_application_insights.application_insights.id
+  description    = "Payments API Call Warning"
+  enabled        = true
+  query = format(<<-QUERY
+  traces
+    | where cloud_RoleName == "%s"
+    | order by timestamp desc
+    | where message contains "[getGPDCheckedReceiptsList] Non-blocking error"
```

Comment:
what's the purpose of this alert ? 
what problem does it mean ?

Reply:
This alert cover two error cases: 

1. When PaymentOption is not found after calling `service.gpd.host`
2. When Receipt is not in an eligible because PaymentsOption and Receipt are not coherent each other

Reply:
could be useful add this comment on alert code